app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -21,8 +21,12 @@ def user_info
     info.merge!(ial2_attributes) if scoper.ial2_scopes_requested? && ial2_data.present?
     info.merge!(x509_attributes) if scoper.x509_scopes_requested?
     info[:verified_at] = verified_at if scoper.verified_at_requested?
-    info[:ial] = Saml::Idp::Constants::AUTHN_CONTEXT_IAL_TO_CLASSREF[identity.ial]
-    info[:aal] = identity.requested_aal_value
+    if identity.vtr.nil?
+      info[:ial] = Saml::Idp::Constants::AUTHN_CONTEXT_IAL_TO_CLASSREF[identity.ial]
+      info[:aal] = identity.requested_aal_value
+    else
+      info[:vtr] = identity.vtr
```

Comment:
And for more context: The `vtr` is the list of acceptable vectors. The `vot` is the vector that describes the transaction that was actually performed.

Reply:
Additional context in spec links:
- [vtr](https://openid.net/specs/openid-igov-openid-connect-1_0.html#RFC8485:~:text=OPTIONAL.%20MUST%20be%20set%20to%20a%20value%20as%20described%20in%20Section%206.1%20of%20Vectors%20of%20Trust%20%5BRFC8485%5D%20vtr%20takes%20precedence%20over%20acr_values.%C2%B6) param in request
- [vot](https://openid.net/specs/openid-igov-openid-connect-1_0.html#RFC8485:~:text=OPTIONAL.%20The%20vector%20value%20as%20specified%20in%20Vectors%20of%20Trust%20%5BRFC8485%5D%20.%20See%20Vectors%20of%20Trust%20(Section%203.5)%20for%20more%20details.%20vot%20takes%20precedence%20over%20acr.) claims in response

Reply:
wait also the `id_token_builder` includes userinfo from this presenter in its final result, should we remove the explicit `vot` logic there since it is here now? (also a good reminder to add the `vtm` claim)

Reply:
vtm claim added.

Reply:
Created `Vot::Parser::Result#expanded_component_values` to encapsulate the expansion code snippet, and used it as appropriate.